doc/ipythonconsole.rst

```diff
@@ -41,10 +41,15 @@ To connect to an external kernel,
 #. Launch an IPython kernel on the local or remote host if one is not already running.
    If using Spyder 3.3.0 or later, you'll need to do so with ``python -m spyder_kernels.console`` (after you've first installed ``spyder-kernels`` on the host with ``<conda/pip> install spyder-kernels``).
    If using a version of Spyder before 3.3.0, ``ipython kernel`` should work to launch the kernel, albeit without certain Spyder-specific features.
-#. Make sure to copy the connection file (:file:`{runtime-dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
+#. Copy the connection file (:file:`{runtime-dir}/kernel-{pid}.json`) to the machine you're running Spyder on (if remote) or note its location (if local).
+   You can get the location of :file:`{runtime-dir}` by running the command ``jupyter_core.paths.jupyter_runtime_dir()`` (make sure you ``import jupyter_core`` first) in the same Python environment as the kernel you launched.
```

Comment:
Please change this paragraph to include the actual code used to get `jupyter_runtime_dir()` instead of describing things with words. I mean, this could be something like:

----

To get the location `runtime-dir` you need to run the following code in an IPython console:

```python
In [1]: from jupyter_core import paths
In [2]: paths.jupyter_runtime_dir()
```

----

We need to be as explicit as possible here.

Reply:
> We need to be as explicit as possible here.

Agreed.

> Please change this paragraph to include the actual code used to get jupyter_runtime_dir()

Well, I did...

> ``jupyter_core.paths.jupyter_runtime_dir()`` (make sure you ``import jupyter_core`` first) in the same Python environment as the kernel you launched.

I changed it to 

> ``import jupyter_core; jupyter_core.paths.jupyter_runtime_dir()`` in an IPython interpreter launched in the same Python environment as the kernel you launched.

to make it even more straightforward and explicit.